src/compare_versions.jl

```diff
@@ -11,9 +11,34 @@ end
 
 function compare_versions(master_version::VersionNumber,
                           head_version::VersionNumber;
-                          allow_unchanged_prerelease::Bool)::Nothing
+                          allow_unchanged_prerelease::Bool,
+                          set_output::Bool,
+                          set_output_io::IO)::Nothing
     if head_version > master_version
         @info("Version number has increased")
+
+        # Set change_type outputs on GH Actions
+        if set_output
+            # 3-digit versions
+            if head_version.major > 0 && master_version.major > 0
+                if head_version.major > master_version.major
+                    set_actions_output(set_output_io, "change_type", "breaking")
+                elseif head_version.minor > master_version.minor
+                    set_actions_output(set_output_io, "change_type", "feature")
+                elseif head_version.patch > master_version.patch
+                    set_actions_output(set_output_io, "change_type", "patch")
+                end
```

Comment:
Technically there are also dev and build version changes, but I personally don't see the point in labelling them. I can add it if you want, though.